src/erlavro.app.src

```diff
@@ -1,7 +1,7 @@
 {application, erlavro,
  [
   {description, "Apache Avro support for Erlang"},
-  {vsn, "1.9.0"},
+  {vsn, "1.9.1"},
   {registered, []},
```

Comment:
I did not change any existing API, only added new.
yes, 1.10.0 makes more sense, but my reason here is that I tagged 1.9.0 too early.
it should have been an rc instead.